contracts/modules/STO/USDTieredSTO.sol

```diff
@@ -137,8 +135,7 @@ contract USDTieredSTO is USDTieredSTOStorage, STO {
      * @param _minimumInvestmentUSD overall minimum investment limit
      */
     function modifyLimits(uint256 _nonAccreditedLimitUSD, uint256 _minimumInvestmentUSD) external onlyOwner {
-        /*solium-disable-next-line security/no-block-members*/
-        require(now < startTime, "STO already started");
+        _isSTOStarted();
```

Comment:
As the check is only one line, It might be better to define it as a function modifier.

Reply:
To save the size, I had to make function instead of modifier. it saves .20 KB